CMakeLists.txt

```diff
@@ -141,6 +141,13 @@ list(APPEND OLIVE_LIBRARIES
 
 # Link PortAudio
 find_package(PortAudio REQUIRED)
+set(CMAKE_REQUIRED_INCLUDES ${PORTAUDIO_INCLUDE_DIRS})
```

Comment:
What does this line do?

Reply:
https://cmake.org/cmake/help/v3.13/module/CheckIncludeFileCXX.html Its there to ensure that CheckIncludeFileCXX checks the correct include dir which the bundled FindPortAudio.cmake should point to.

Reply:
No, the `set(CMAKE_REQUIRED_INCLUDES ${PORTAUDIO_INCLUDE_DIRS})` line

Reply:
Ah I see